### PR TITLE
Implemented a test case for biceps5-4-7_11.

### PR DESCRIPTION
**Pull request by ben-Draeger**
The headline says it all.

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Adherence to coding conventions
  * [x] Pull Request Assignee
  * [x] Reviewer
* Adherence to javadoc conventions
  * [x] Pull Request Assignee
  * [x] Reviewer
* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* config update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* SDCcc executable ran against a test device (if necessary)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java [hunk 1]
```diff
@@ -752,4 +752,58 @@ void testMetricStatusManipulationSETActivationStateNOTRDYBadSecondManipulationFa
...
+        // let setMetricStatus fail
+        when(mockManipulations.setMetricStatus(
+                        any(String.class), any(MetricCategory.class), any(ComponentActivation.class)))
+                .thenReturn(ResponseTypes.Result.RESULT_FAIL);
+
+        assertFalse(ManipulationPreconditions.MetricStatusManipulationSETActivationStateFAIL.manipulation(injector));
+
+        assertTrue(
+                testRunObserver.isInvalid(),
+                "Test run should not have been invalidated. Reason(s): " + testRunObserver.getReasons());
```
Comment (belagertem) [inline sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java:804]: The error message should state that the test run should not have been valid.

File: sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java [hunk 2]
```diff
@@ -752,4 +752,58 @@ void testMetricStatusManipulationSETActivationStateNOTRDYBadSecondManipulationFa
...
+
+        // let setComponentActivation fail
+        when(mockManipulations.setComponentActivation(any(String.class), any(ComponentActivation.class)))
+                .thenReturn(ResponseTypes.Result.RESULT_FAIL);
+
+        assertFalse(ManipulationPreconditions.MetricStatusManipulationSETActivationStateFAIL.manipulation(injector));
+
+        assertTrue(
+                testRunObserver.isInvalid(),
+                "Test run should not have been invalidated. Reason(s): " + testRunObserver.getReasons());
```
Comment (belagertem) [inline sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java:785]: The error message should state that the test run should not have been valid.

File: sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java [hunk 3]
```diff
@@ -752,4 +752,58 @@ void testMetricStatusManipulationSETActivationStateNOTRDYBadSecondManipulationFa
         verify(mockManipulations).setComponentActivation(metricStateHandle, startActivationState);
         verify(mockManipulations).setMetricStatus(metricStateHandle, metricCategory, activationState);
     }
+
+    @Test
+    @DisplayName(
+            "testMetricStatusManipulationSETActivationStateFAIL: Set ActivationState " + "of all SET-Metrics to FAIL.")
```
Comment (belagertem) [inline sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java:758]: Why the concatenation?

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - test for biceps:5-4-7_3
 - test for biceps:5-4-7_5
 - test for biceps:5-4-7_7
+- test for biceps:5-4-7_11
+- support for SystemErrorReport to test case glue:R0036_0
```
Comment (belagertem) [inline CHANGELOG.md:15]: not part of this pr

File: sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelStatePartTestTest.java [hunk 5]
```diff
@@ -3412,6 +3412,325 @@ public void testRequirement54710BadMultipleReportsInInterval() throws Exception
...
+        final var result = ResponseTypes.Result.RESULT_SUCCESS;
+        final var methodName = Constants.MANIPULATION_NAME_SET_METRIC_STATUS;
+        final List<Pair<String, String>> parameters = List.of(
+                new ImmutablePair<>(Constants.MANIPULATION_PARAMETER_HANDLE, SET_METRIC_HANDLE),
+                new ImmutablePair<>(Constants.MANIPULATION_PARAMETER_METRIC_CATEGORY, MetricCategory.SET.value()),
+                new ImmutablePair<>(
+                        Constants.MANIPULATION_PARAMETER_COMPONENT_ACTIVATION, ComponentActivation.FAIL.value()));
+        messageStorageUtil.addManipulation(storage, TIMESTAMP_START, TIMESTAMP_FINISH, result, methodName, parameters);
+
+        // activation state should be OFF
```
Comment (belagertem) [inline sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelStatePartTestTest.java:3657]: should be fail

**Review by belagertem — CHANGES_REQUESTED** [review]

File: sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java (hunk 3 above)
Comment (ben-Draeger) [reply sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java:758]: The string was supposed to be in 2 lines. Spotless reformatted it back into one line, which indeed looks a bit silly.
I will remove this.

File: CHANGELOG.md (hunk 4 above)
Comment (ben-Draeger) [reply CHANGELOG.md:15]: The last line was merged in from main. The only thing I did was to reorder the lines.

File: sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java (hunk 1 above)
Comment (ben-Draeger) [reply sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java]: Done.

File: sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java (hunk 2 above)
Comment (ben-Draeger) [reply sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java]: Done.

File: sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java (hunk 3 above)
Comment (ben-Draeger) [reply sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java]: Done.

File: CHANGELOG.md (hunk 4 above)
Comment (ben-Draeger) [reply CHANGELOG.md:17]: I merged in the master again. So be careful, the diff might look weird again...

File: sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelStatePartTestTest.java (hunk 5 above)
Comment (ben-Draeger) [reply sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelStatePartTestTest.java]: Corrected.

**Review by belagertem — APPROVED** [review]
